drivers/adc/adc_npcx.c

```diff
@@ -150,6 +153,19 @@ static void adc_npcx_pm_policy_state_lock_put(struct adc_npcx_data *data)
 }
 #endif
 
+static inline void adc_npcx_config_channels(const struct device *dev, uint32_t channels)
+{
+	const struct adc_npcx_config *config = dev->config;
+	struct adc_reg *const inst = HAL_INSTANCE(dev);
+
+	inst->ADCCS = channels & BIT_MASK(16);
```

Comment:
maybe add a define for 16? NPCX_ADC_MAX_CHANNEL_COUNT or something

Reply:
Done.